performance/tests/Button.test.js

```diff
@@ -4,17 +4,28 @@ const {getFileName} = require('../utils');
 const TestResults = require('../TestResults');
 
 describe('Button', () => {
+	let browser, page;
+
+	beforeEach(async () => {
+		browser = await puppeteer.launch({headless: true});
+
+		page = await browser.newPage();
+		await page.setViewport({
+			width: 1920,
+			height: 1080
+		});
+
+		const client = await page.target().createCDPSession();
+		await client.send('Emulation.setCPUThrottlingRate', {rate: 6});
```

Comment:
6 is a reasonable number if tested in a PC environment. Since we don't have any other test environments in the near future, I didn't bother implementing it as a variable. I'd rather create a follow-up ticket rather than fix here in a "clean up" ticket.

Reply:
That makes sense